components/clarinet-format/tests/golden-intended/nested_map.clar

```diff
@@ -13,19 +13,18 @@
       stx-burn: u0,
       owner: send-to,
     })
-  (print
-    {
-      topic: "new-name",
-      owner: send-to,
-      name: {
+  (print {
+    topic: "new-name",
+    owner: send-to,
+    name: {
+      name: name,
+      namespace: namespace,
+    },
+    id: id-to-be-minted,
+    properties: (map-get? name-properties
+      {
         name: name,
         namespace: namespace,
-      },
-      id: id-to-be-minted,
-      properties: (map-get? name-properties
-        {
-          name: name,
-          namespace: namespace,
-        }),
-    })
+      }),
```

Comment:
The closing `)` of `map-get?` isn"t on the same indentation leval as the opening one.
Should be
```clarity
  properties: (map-get? name-properties
    {
      name: name,
      namespace: namespace,
    }
  )
```
or even better imo
```clarity
```clarity
  properties: (map-get? name-properties {
    name: name,
    namespace: namespace,
  })
```